src/app.js

```diff
@@ -8,6 +8,8 @@
 import express from 'express';
 import morgan from 'morgan';
 import { config as hathorLibConfig, wallet as oldWalletUtils } from '@hathor/wallet-lib';
+// TODO: Import from '@hathor/wallet-lib/lib/config' as soon as it's available
+import { SWAP_SERVICE_MAINNET_BASE_URL, SWAP_SERVICE_TESTNET_BASE_URL } from './constants';
```

Comment:
The next version of the Wallet Lib will have these constants exported ( PR HathorNetwork/hathor-wallet-lib#471 ), so we'll fetch if from there instead duplicating them here.

For now, this is following the pattern used by the Wallet Service of having the URLs on the client application.
https://github.com/HathorNetwork/hathor-wallet/blob/18c8314ea3df6b071d93277a2a797b52948ab455/src/constants.js#L237-L238